### PR TITLE
add assistant beta header to make it possible to use assistants

### PR DESCRIPTION
**Pull request by frithjof**
### What:

- [ ] Bug Fix
- [ ] New Feature

### CODE REVIEW

**Review by GromNaN — APPROVED** [review]
It's also in the laravel brdige.
https://github.com/openai-php/laravel/blob/cdcf65d19a9a847cfdf249e7b1b6897ea6e9b650/src/ServiceProvider.php#L36

**Comment by DylanDelobel** [comment]
This would solve https://github.com/openai-php/symfony/issues/15

@frithjof Could you fix coding style on your PR ? :)
```diff
  -        ->call('withHttpHeader', ['OpenAI-Beta' , 'assistants=v1'])
  +        ->call('withHttpHeader', ['OpenAI-Beta', 'assistants=v1'])
  ```

File: src/Resources/config/services.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
         ->set(Factory::class)
         ->factory([OpenAI::class, 'factory'])
         ->call('withHttpClient', [service('openai.http_client')])
+        ->call('withHttpHeader', ['OpenAI-Beta' , 'assistants=v1'])
```
Comment (GromNaN) [inline src/Resources/config/services.php:20]:
```suggestion
        ->call('withHttpHeader', ['OpenAI-Beta', 'assistants=v1'])
```

**Comment by GromNaN** [comment]
Thank you @frithjof

**Comment by nunomaduro** [comment]
@gehrisandro can you confirm this change is ok?

**Comment by DylanDelobel** [comment]
@nunomaduro Just tried
```php
$result = $client->assistants()->list();
dd($result);
```
All good 👍

**Comment by gehrisandro** [comment]
@nunomaduro Looks good to me.

**Pull request merged**